### DISCUSSION TITLE
Release 0.54.0 labelling wrong year?

### DISCUSSION THREAD

**Discussion by ulisse31** [prometheus-operator/prometheus-operator#4559] [General]
Hello!
I see in the release page for the latest release: `0.54.0 / 2021-01-26`
If that is referencing the release date, then it should be 2022 :)

I hope I'm not missing something, cheers
Andrea

**Comment by simonpasquier** [comment]
you're correct, thanks for pointing out!

**Comment by simonpasquier** [comment]
And the CHANGELOG.md would need a fix too :)
